### PR TITLE
Contributing guidelines added. Issue #7

### LINKED ISSUE #7: CONTRIBUTING.md is needed.
I would like to add contributing guidelines to the repo. Kindly let me know if I can take this up. Any suggestions are welcome. Thank you.

### PR DESCRIPTION
**Pull request by nishihere19**
**Description**

Added contributing guidelines.

**Related Issue** 
- issue goes here

Fixes #7 

- [ ] Code
- [ ] User Interface
- [ ] New Feature
- [x] Documentation
- [ ] Testing

**Code/Quality Assurance Only**

- Bug fix (non-breaking change which fixes an issue)
- This change requires a documentation update (software upgrade on readme file)

**How Has This Been Tested?**

**Additional Info (OPTIONAL)**

**Checklist**

- [ ] My code follows the code style of this project.
- [ ] My UI is responsive
- [ ] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [x] My changes generate no new warnings
- [x] The title of my pull request is a short description of the requested changes.

**Screenshots**
   Original                 |   Updated
  :------------------------:|---------------------:
  ** original screenshot ** | ** updated screenshot **

### CODE REVIEW

**Review by NeelParihar — COMMENTED** [review]
Try to ignore commiting changes of package-lock.json

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+
+# :bookmark:Code advice
+
+Kindly maintain a clean code with proper comments.
+
+Commit messages should be written carefully and describe the changes made.
+
+Create separate PR for a single feature update/bug fix.
+
```
Comment (NeelParihar) [inline CONTRIBUTING.md:36]: Add some space after the emojis as they are overlapping with other text

Screenshot 2021-03-16 at 4 57 45 PM

**Review by NeelParihar — CHANGES_REQUESTED** [review]

File: CONTRIBUTING.md (hunk 1 above)
Comment (nishihere19) [reply CONTRIBUTING.md:36]: Sure!

File: .gitignore [hunk 2]
```diff
@@ -35,4 +35,5 @@ lerna-debug.log*
 
 
 .env
-service.json
\ No newline at end of file
+service.json
+package.lock.json
```
Comment (Amulya-coder) [inline .gitignore:39]: Try to avoid these changes.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+>git merge upstream/(your_branch_name)
+
+:alien: Push your changes to your branch.
+
+:wrench: Make a pull request from your branch.
+
+:heavy_plus_sign: Before submitting a PR, check your code well and explain your work properly in the PR description.
+
+
+
```
Comment (Amulya-coder) [inline CONTRIBUTING.md:28]: remove the extra space from here!

**Review by Amulya-coder — CHANGES_REQUESTED** [review]

File: .gitignore (hunk 2 above)
Comment (Abhishek-kumar09) [inline .gitignore:39]: Please revert this.

**Review by Abhishek-kumar09 — CHANGES_REQUESTED** [review]

**Comment by nishihere19** [comment]
Kindly review it once again, if the changes are implemented properly.

**Comment by Amulya-coder** [comment]
@nishihere19 all looks good but the checks are failing I don't know why, I can't approve until all the checks are passed.
Cc:- @Abhishek-kumar09

**Comment by nishihere19** [comment]
There was an unused variable in app.module.ts which was failing the test case. I only removed that.

**Review by Amulya-coder — APPROVED** [review]
LGTM!!

**Comment by Abhishek-kumar09** [comment]
Thanks @nishihere19 for the PR, thanks @Amulya-coder for the review.

**Pull request merged**